### PR TITLE
fix issue where Scrollpane cannot scroll properly

### LINKED ISSUE #56: Improve UI of help command

### PR DESCRIPTION
**Pull request by weiliann**
Fixes #56 

1. Fix a bug where the help window cannot scroll
2. Add view command to the help list

### CODE REVIEW

**Review by shizy — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -36,13 +36,16 @@ public class HelpWindow extends UiPart<Stage> {
             + "5. Find by Name\n"
```
Comment (yyihaoc) [inline src/main/java/seedu/address/ui/HelpWindow.java]: can find by role as well

**Review by yyihaoc — COMMENTED** [review]
LGTM

**Review by yyihaoc — APPROVED** [review]
LGTM!

**Pull request merged**